docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
     tty: true
     volumes:
       - "/path/to/resources/folder:/config"
+      - "/path/to/txData/folder:/txData"
```

Comment:
As this compose file needs to cover all use cases, a comment explaining this is only for txAdmin would be good